.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+- 2.2.9
```

Comment:
@Altech Do you think we should support Ruby 2.2? I don't think so, because it is shortly EOL.

Reply:
And we are using lonely operator. https://travis-ci.org/Altech/red_blocks/jobs/316922306